src/pages/CommitDetailPage/CommitBundleAnalysis/CommitBundleAnalysis.tsx

```diff
@@ -59,13 +61,22 @@ const BundleContent: React.FC<BundleContentProps> = ({ bundleCompareType }) => {
 
 const CommitBundleAnalysis: React.FC = () => {
   const { provider, owner, repo, commit: commitSha } = useParams<URLParams>()
+  const { data: overview } = useRepoOverview({ provider, owner, repo })
   const { data: commitPageData } = useCommitPageData({
     provider,
     owner,
     repo,
     commitId: commitSha,
   })
 
+  useEffect(() => {
+    if (overview?.bundleAnalysisEnabled && overview?.coverageEnabled) {
+      metrics.increment('commit_detail_page.bundle_dropdown.opened', 1)
```

Comment:
Does the user actually open a dropdown in this case? It looks like we are just checking repo overview to see if they have features enabled.

Reply:
So if both of these are true, and this component is rendered we know for a fact that the dropdown was opened. There's not real great way that the dropdown exposes itself to check and see if the dropdown was opened or closed.